Airline_Web_API/Airline_Web_API/Controllers/FlightsController.cs

```diff
@@ -22,17 +22,26 @@ public FlightsController(AirlineContext context)
 
         // GET: api/flights
         [HttpGet]
-        public async Task<ActionResult<IEnumerable<Flight>>> GetFlights([FromQuery] int departure, int arrival, DateTime date)
+        public async Task<ActionResult<IEnumerable<Flight>>> GetFlights([FromQuery] int departure, int arrival, DateTime date, int amount)
```

Comment:
Please review naming. 'amount' of what exactly? Also arrival and departure with type int, what are they?